src/videoCallWindow/ipc.ts

```
@@ -296,11 +300,14 @@ export const startVideoCallWindowHandler = (): void => {
     // jitsiBridge's ipcRenderer.on listener fires correctly.
     ipcMain.once(
       'video-call-window/screen-sharing-source-responded',
-      (_event, sourceId: string | null) => {
+      (
+        _event,
+        payload: string | null | ScreenSharingSelectionPayload
+      ) => {
```

ESLint

[error] Replace `⏎········_event,⏎········payload:·string·|·null·|·ScreenSharingSelectionPayload⏎······` with `_event,·payload:·string·|·null·|·ScreenSharingSelectionPayload`

(prettier/prettier)